.github/workflows/build.yml

```diff
@@ -107,3 +107,17 @@ jobs:
         with:
           name: rpm-artifacts
           path: ${{ env.ARTIFACTS_DIR}}
+
+  e2e:
+    needs: rpmbuild 
+    runs-on: ubuntu-latest
```

Comment:
I failed all of my tests to sync jobs from 2 workflows from different yamls.
e2e is running on maim if i trigger it manually.

Any idea is welcome
@ygalblum @engelmi

Reply:
Something that I have done in the past is inverting the dependencies. So instead of telling the e2e workflow to run once the build workflow is completed, you could directly invoke the e2e workflow from the build workflow.

This can be done by something like this:

**.github/workflows/e2e.yml**
```yml
name: Integration Testing
on: [workflow_call] 
...
```

**.github/workflows/deploy.yml**
```yml
name: Build
jobs:
  rpmbuild:
    ...
  e2e:
    uses: ./.github/workflows/e2e.yml  
    needs: rpmbuild
```

Reply:
Thanks, 
I did not try this one,